maestro-cli/src/main/java/maestro/cli/analytics/Analytics.kt

```diff
@@ -73,6 +77,13 @@ object Analytics {
     fun maybeAskToEnableAnalytics() {
         if (hasRunBefore) return
 
+        // Fix for https://github.com/mobile-dev-inc/maestro/issues/1846
+        if (CiUtils.getCiProvider() != null) {
```

Comment:
Let's maybe give DISABLE_ANALYTICS_ENV_VAR false by default if its CI?

Reply:
We only need analytics if user is sourcing from CLI

Reply:
Are you sure? We set `uuid` to name of the CI service if we're running on CI:

https://github.com/mobile-dev-inc/maestro/blob/21ae6eb6eae669a397174dc69bedef203384413e/maestro-cli/src/main/java/maestro/cli/analytics/Analytics.kt#L191-L193

which then gets uploaded to our backend:

https://github.com/mobile-dev-inc/maestro/blob/21ae6eb6eae669a397174dc69bedef203384413e/maestro-cli/src/main/java/maestro/cli/analytics/Analytics.kt#L167

I think it can be useful knowledge to know what CI service is most used*, so we would have some data if we will have to build integrations with CIs in the future.

---

*A problem here may be that we don't have UUIDs of specific CI installations. We would need to have separate "UUID" and "CI" values in the AnalyticsReport.

https://github.com/mobile-dev-inc/maestro/blob/21ae6eb6eae669a397174dc69bedef203384413e/maestro-cli/src/main/java/maestro/cli/analytics/Analytics.kt#L204-L220

Otherwise analytics from various runs on CI get merged together.